### ISSUE TITLE
How To file

### ISSUE THREAD

**Issue by stochasticsdrj** [jbustamante35/hypoquantyl#77]
Good day Julian,
The How To file is not populated.  I would be interested to analyze some etiolated wheat seedlings of mine grown under different temperatures for growth rate, if possible.  Thanks, Julian (also my name)

**Comment by jbustamante35** [comment]
Hello Julian, thanks for your comment! I honestly didn't expect anyone to be interested in this just yet, so several files are incomplete. I'm sorry to say, but I'm still actively working on this for my PhD thesis and it isn't quite ready for general use. I'm able to do kinematics on my own experiments, but it is nowhere near user-friendly. 

However, I am interested in your experiments. I've currently trained my program on grayscale images of _Arabidopsis_ seedlings, but I intend on broadening it's use for many different plant species and image types. Would you be willing to send me some example images you were planning on analyzing (only if you're comfortable sending them)? It'd be extremely helpful for me to get a sense of what experiments other users would want to analyze.

**Comment by stochasticsdrj** [comment]
Hi Julian, thanks for your quick response.  I completely understand.  I will share some images shortly, no problem.  I started using KymoRod, which works well to an extent, however from what I can ascertain it can only output what they call "Relative Elemental Growth Rate" - I am also interested in an absolute measure e.g. um/min or mm/hr.  Unfortunately I don't have the programming savvy to 'peak under the hood' to figure out how they are using/translating raw pixels.

**Comment by stochasticsdrj** [comment]
Hi Julian, did you get the email I sent you with a folder of images?

**Comment by jbustamante35** [comment]
Yes I did! Sorry I wasn't able to reply yesterday, but I was just about to respond to your email.
